### PR TITLE
JSON LD changes

### PR DESCRIPTION
**Pull request by tntra-dhananjay**
Created JSON-LD files at https://github.com/spdci/schemas/

Added reference to JSON-LD in description

### CODE REVIEW

File: release/yaml/ibr_api_v1.0.0.yaml [hunk 1]
```diff
@@ -2251,7 +2254,7 @@ components:
         name:
           type: string
           description: |
-            @context: "https://example.org/schema/Attribute" <br>
+            @context: "https://spdci.github.io/schemas//Attribute" <br>
```
Comment (naftis) [inline release/yaml/ibr_api_v1.0.0.yaml]: a minor search + replace problem repeats with the spdci.github.io/schemas// url

**Comment by naftis** [comment]
@vvujjini @tntra-dhananjay Should we tag new incoming changes as 1.0.1 to allow Dasun and me to work with 1.0.0?

**Comment by vvujjini** [comment]
> @vvujjini @tntra-dhananjay Should we tag new incoming changes as 1.0.1 to allow Dasun and me to work with 1.0.0?

@naftis agree. 

However most of the changes proposed so far are to do with schema entity referencing and NOT to do with any core of the API envelope specs. It will be good you focus on 1.0.0 tag and towards the end we can do a diff between 1.0.0 tag and latest code from the branch and decide to tag this 1.0.1 or continue to make spec edits for next agreed stable point release.

File: release/yaml/ibr_api_v1.0.0.yaml (hunk 1 above)
Comment (tntra-dhananjay) [reply release/yaml/ibr_api_v1.0.0.yaml]: This will be replaced with domain - https://schema.spdci.org once domain available to me.

**Comment by tntra-dhananjay** [comment]
> > @vvujjini @tntra-dhananjay Should we tag new incoming changes as 1.0.1 to allow Dasun and me to work with 1.0.0?
> 
> @naftis agree.
> 
> However most of the changes proposed so far are to do with schema entity referencing and NOT to do with any core of the API envelope specs. It will be good you focus on 1.0.0 tag and towards the end we can do a diff between 1.0.0 tag and latest code from the branch and decide to tag this 1.0.1 or continue to make spec edits for next agreed stable point release.

Created new tag 1.0.0 >> https://github.com/spdci/standards/releases/tag/1.0.0

File: src/extensions/mosip/MOSIPVerifiableCredential.yaml [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
-      postalCode:
+      postal_code:
         type: string
         example: "14022"
-      fullName:
+      full_name:
         type: string
         example: "Thirteen Mosip"
-      dateOfBirth:
+      dateofbirth:
```
Comment (naftis) [inline src/extensions/mosip/MOSIPVerifiableCredential.yaml:25]: date_of_birth rather?

File: src/common/schema/SearchSort.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ properties:
   attribute_name:
     type: string
     description: |
-      @context: "https://example.org/schema/Attribute" <br>
+      @context: "https://standards.spdci.org/schemas//Attribute" <br>
```
Comment (naftis) [inline src/common/schema/SearchSort.yaml]:
```suggestion
      @context: "https://standards.spdci.org/schemas/Attribute" <br>
```

File: src/common/schema/Meta.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 type: object
 description: |
-  @context: "https://example.org/schema/Meta" <br>
+  @context: "https://standards.spdci.org/schemas//Meta" <br>
```
Comment (naftis) [inline src/common/schema/Meta.yaml]:
```suggestion
  @context: "https://standards.spdci.org/schemas/Meta" <br>
```

File: src/common/schema/IdentifierTypeValue.yaml [hunk 5]
```diff
@@ -4,7 +4,7 @@ properties:
   type:
     type: string
     description: |
-      @context: "https://example.org/schema/IdType" <br>
+      @context: "https://standards.spdci.org/schemas//IdType" <br>
```
Comment (naftis) [inline src/common/schema/IdentifierTypeValue.yaml]:
```suggestion
      @context: "https://standards.spdci.org/schemas/IdType" <br>
```

File: src/common/schema/IdentifierType.yaml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 type: string
 description: |
-  @context: "https://example.org/schema/IdType" <br>
+  @context: "https://standards.spdci.org/schemas//IdType" <br>
```
Comment (naftis) [inline src/common/schema/IdentifierType.yaml]:
```suggestion
  @context: "https://standards.spdci.org/schemas/IdType" <br>
```

File: src/common/schema/ExpTemplate.yaml [hunk 7]
```diff
@@ -14,7 +14,7 @@ properties:
   value:
     type: object
     description: |
-      @context: "https://example.org/schema/QueryExpression" <br>
+      @context: "https://standards.spdci.org/schemas//QueryExpression" <br>
```
Comment (naftis) [inline src/common/schema/ExpTemplate.yaml]:
```suggestion
      @context: "https://standards.spdci.org/schemas/QueryExpression" <br>
```

File: src/common/schema/ExpTemplate.yaml [hunk 8]
```diff
@@ -4,7 +4,7 @@ properties:
   type:
     type: string
     description: |
-      @context: "https://example.org/schema/QueryType" <br>
+      @context: "https://standards.spdci.org/schemas//QueryType" <br>
```
Comment (naftis) [inline src/common/schema/ExpTemplate.yaml]:
```suggestion
      @context: "https://standards.spdci.org/schemas/QueryType" <br>
```

File: src/common/schema/ExpPredicate.yaml [hunk 9]
```diff
@@ -4,7 +4,7 @@ properties:
   attribute_name:
     type: string
     description: |
-      @context: "https://example.org/schema/QueryAttributes" <br>
+      @context: "https://standards.spdci.org/schemas//QueryAttributes" <br>
```
Comment (naftis) [inline src/common/schema/ExpPredicate.yaml]:
```suggestion
      @context: "https://standards.spdci.org/schemas/QueryAttributes" <br>
```

File: src/extensions/ibr/Person.yaml [hunk 10]
```diff
@@ -3,11 +3,9 @@ description: |
   1. Attributes of a person to create fetch records, create verifiable credentials or use in search criteria.
   3. Allowes Country/Registry specific implementation extensions using Attribute Name/Value pairs.
 properties:
-  identifier_type:
-    $ref: IdentifierType.yaml
   identifier:
-    type: string
-    description: Value of the identifier
+    $ref: IdentifierTypeValue.yaml
+    description: "Information of identifer of the person"
```
Comment (naftis) [inline src/extensions/ibr/Person.yaml]:
```suggestion
    description: "Information of identifier of the person"
```

File: src/extensions/ibr/Person.yaml [hunk 11]
```diff
@@ -30,17 +28,17 @@ properties:
...
   programms:
     type: array
-    description: "TO_DO"
+    description: "Details of the programmes the person is enrolled in"
     items:
       $ref: Programme.yaml
   benefits:
     type: array
-    description: "TO_DO"
+    description: "Benifits received by the person"
```
Comment (naftis) [inline src/extensions/ibr/Person.yaml]:
```suggestion
    description: "Benefits received by the person"
```

File: src/extensions/mosip/MOSIPVerifiableCredential.yaml (hunk 2 above)
Comment (naftis) [inline src/extensions/mosip/MOSIPVerifiableCredential.yaml]:
```suggestion
      date_of_birth:
```
or
```suggestion
      birthdate:
```
or is this MOSIP related so should be dateofbirth?

File: src/extensions/mosip/MOSIPVerifiableCredential.yaml [hunk 12]
```diff
@@ -1,43 +1,43 @@
...
         type: string
         example: "2022"
       province:
         $ref: LanguageValueList.yaml
       phone:
         type: string
         example: "9898989898"
-      addressLine1:
+      addressline1:
```
Comment (naftis) [inline src/extensions/mosip/MOSIPVerifiableCredential.yaml:33]:
```suggestion
      address_line_1:
```

File: src/extensions/mosip/MOSIPVerifiableCredential.yaml [hunk 13]
```diff
@@ -1,43 +1,43 @@
...
         $ref: LanguageValueList.yaml
-      addressLine2:
+      addressline2:
```
Comment (naftis) [inline src/extensions/mosip/MOSIPVerifiableCredential.yaml:35]:
```suggestion
      address_line_2:
```

**Review by naftis — COMMENTED** [review]
Great work, just some minor typos/suggestions probably worth fixing

**Review by vvujjini — APPROVED** [review]
Merging
